src/suggestions/suggestions-controller.ts

```diff
@@ -772,16 +807,14 @@ export class MenuController {
                     );
 
                     this.insertOptionIntoMenu(option, menu);
-
-                    if (option.text === focusedOptionText) {
```

Comment:
This kept the currently focused option in focus as the user was typing. This is a bit difficult to do now that the menu has a window of options it displays. This requires `topOptionIndex` and `bottomOptionIndex` to be constantly maintained on each focus. I don't think this functionality is very important and because we are beautifying the text with `TextEnhance` this case won't even be true anymore.

However, if we ever want to get this functionality back all we have to do is compare the pre-TextEnhance `option.text` to `focusedOptionText` and then calculate the correct values for the option boundaries based on the position of this option in the array of options.